Tests/TestingApps/PaywallsTester/PaywallsTester/SimpleApp.swift

```diff
@@ -64,6 +71,18 @@ extension SimpleApp {
             return nil
         }
 
-        return IdentifiableString(id: paywallID)
+        let showIntroOffer = params.first(where: { $0.name == "io" })?.value.flatMap { value -> IntroEligibilityStatus in
+            if value == "1" {
+                return .eligible
+            } else if value == "0" {
+                return .ineligible
+            } else if value == "n" {
+                return .noIntroOfferExists
+            } else {
+                return .unknown
+            }
+        } ?? .unknown
```

Comment:
feels like this is complex / specific enough to be in its own function? 
I'm also curious about the naming for these - we're going with the shortest possible values because these are deeplinks? If so, maybe we can make them constants or something for readability rather than magic strings? 

I'm kinda torn on the suggestion because adding indirection might not necessarily make this easier to read in any case, given that a simple if / else block here is still fairly clear

Reply:
oh I like that. Yeah, I kinda figured we're going for min length for deeplink purposes, sounds good